README.md

```diff
@@ -24,6 +24,28 @@ For Beta: ("beta")
 ```
 And for Production use: `@string/kinecosystem_environment_production` as value. 
 
+## Enable Auto backup ##
+Android Auto Backup automatically backs up apps data that target and run on Android 6.0 (API level 23) and above.
+Enabling auto backup will provide better experience when user uninstall and reinstall the app.
+No extra action is needed, if auto backup is enabled and ```android:fullBackupContent``` is not defined.
+Otherwise the app module AndroidManifest.xml should include these attributes.
+```xml
+ <application
+        android:allowBackup="true"
+        tools:replace="android:allowBackup"
+        android:fullBackupContent="@xml/backup_rules">  
+</application>
+```
+and ```backup_rules.xml``` should include 
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<full-backup-content>
+    <include domain="sharedpref" path="KinKeyStore_kinecosystem_store.xml"/>
```

Comment:
Do we need only to keep this file?
Shouldn't we keep also blockchain source file ?